DATA/UNIVERSE/SYSTEMS/NOMAD_PROXY/nomad_proxy.ini

```diff
@@ -12,4 +12,60 @@ nickname = Nomad_Proxy_Arrival
 pos = 0, 0, 0
 visit = 128
 Archetype = airlock_gate_invisible
-goto = Nomad_Proxy, Nomad_Proxy_Arrival, gate_tunnel_bretonia
\ No newline at end of file
+goto = Nomad_Proxy, Nomad_Proxy_Arrival, gate_tunnel_bretonia
+
+[Object]
+nickname = Nomad_Proxy_to_St01
+pos = 0, 101000, 0
+visit = 128
+Archetype = airlock_gate_invisible
+goto = St01, St01_Proxy_Arrival, gate_tunnel_bretonia
+
+[Object]
+nickname = Nomad_Proxy_to_St02
+pos = 0, 102000, 0
+visit = 128
+Archetype = airlock_gate_invisible
+goto = St02, St02_Proxy_Arrival, gate_tunnel_bretonia
+
+[Object]
+nickname = Nomad_Proxy_to_St03
+pos = 0, 103000, 0
+visit = 128
+Archetype = airlock_gate_invisible
+goto = St03, St03_Proxy_Arrival, gate_tunnel_bretonia
+
+[Object]
+nickname = Nomad_Proxy_to_St03b
+pos = 0, 104000, 0
+visit = 128
+Archetype = airlock_gate_invisible
+goto = St03b, St03b_Proxy_Arrival, gate_tunnel_bretonia
+
+[Object]
+nickname = Nomad_Proxy_to_St02c
+pos = 0, 105000, 0
+visit = 128
+Archetype = airlock_gate_invisible
+goto = St02c, St02c_Proxy_Arrival, gate_tunnel_bretonia
+
+[Object]
+nickname = Nomad_Proxy_to_Start
```

Comment:
I would see this one as part of "Independent", but it is not very important. Theoretically it could be even entirely left behind. In fact, funny enough, it will not be usable unless some beams to the start system via admin commands. But why not.

Reply:
Next time we do testing in the start system we dont want to be surprised if something doesnt work as expected ;)